Terminal.Gui/View/View.Command.cs

```diff
@@ -0,0 +1,320 @@
+﻿#nullable enable
+using System.ComponentModel;
+
+namespace Terminal.Gui;
+
+public partial class View // Command APIs
+{
+    #region Default Implementation
+
+    /// <summary>
+    ///     Helper to configure all things Command related for a View. Called from the View constructor.
+    /// </summary>
+    private void SetupCommands ()
+    {
+        // Enter - Raise Accepted
+        AddCommand (Command.Accept, RaiseAccepting);
+
+        // HotKey - SetFocus and raise HandlingHotKey
+        AddCommand (Command.HotKey,
+                    () =>
+                    {
+                        if (RaiseHandlingHotKey () is true)
+                        {
+                            return true;
+                        }
+
+                        SetFocus ();
+
+                        return true;
+                    });
+
+        // Space or single-click - Raise Selecting
+        AddCommand (Command.Select, (ctx) =>
+                                    {
+                                        if (RaiseSelecting (ctx) is true)
+                                        {
+                                            return true;
+                                        }
+
+                                        if (CanFocus)
+                                        {
+                                            SetFocus ();
+
+                                            return true;
+                                        }
+
+                                        return false;
+                                    });
+    }
+
+    /// <summary>
+    ///     Called when the user is accepting the state of the View and the <see cref="Command.Accept"/> has been invoked. Calls <see cref="OnAccepting"/> which can be cancelled; if not cancelled raises <see cref="Accepting"/>.
+    ///     event. The default <see cref="Command.Accept"/> handler calls this method.
```

Comment:
I agree `Accepting` alone feels weird. 

I considered having two events but rejected it because I couldn't think of any use case where `Accepting` wouldn't work fine.

"Don't add an API unless there's a known use-case for it."

Reply:
> "Don't add an API unless there's a known use-case for it."

I agree, but thus the `Accept` name is much more appropriate for the event name, like it was before, aligned with the `Command.Accept` name.